packages/client/src/components/block-defaults/upload-block/config.tsx

```diff
@@ -45,6 +74,17 @@ export const config: BlockConfig<UploadBlockDef> = {
                         <UploadSettings id={id} label="Value" path={'value'} />
                     ),
                 },
+                {
+                    description: 'Extensions',
+                    render: ({ id }) => (
+                        <SelectSettings
+                            id={id}
+                            label="Extensions"
+                            path={'extensions'}
+                            extensions={FileTypes}
```

Comment:
Can we rename this to options? Needs to be genericized as a component we can use to pass other options that we have stored as constants, will not always be extensions.

We also may have a Select Block Settings component already, if that is the case please use that one over this.  It would be in `/client/src/components/block-settings

Reply:
updated extensions to Option to make it more generic for reusability

SelectInputSettings is designed for single option selection